kotlin/src/main/kotlin/com/linecorp/armeria/internal/common/kotlin/FlowCollectingPublisher.kt

```diff
@@ -39,19 +39,17 @@ internal class FlowCollectingPublisher<T>(
 ) : Publisher<T> {
     @OptIn(DelicateCoroutinesApi::class)
     override fun subscribe(s: Subscriber<in T>) {
-        val delegate = DefaultStreamMessage<T>()
+        val delegate = StreamMessage.streaming<T>()
         val job = GlobalScope.launch(context) {
             try {
                 flow.collect {
                     delegate.write(it!!)
                     delegate.whenConsumed().await()
                 }
             } catch (e: Throwable) {
-                if (!delegate.tryClose(e)) {
-                    // Delegate to coroutine's exception handling mechanism,
-                    // which ignores CancellationException.
-                    throw e
-                }
+                // Delegate to coroutine's exception handling mechanism,
+                // which ignores CancellationException.
```

Comment:
```suggestion
```
We don't delegate anymore so let's just remove this comment. 😄

Reply:
✅ updated!